db/migrate/20220321091220_add_personal_owner_to_teams.rb

```diff
@@ -0,0 +1,25 @@
+class AddPersonalOwnerToTeams < ActiveRecord::Migration[6.1]
+  def up
+    add_column :teams, :type, :string, null: false
+
+    Team.all.find_each do |team|
+      team.update!(type: Team::Shared.sti_name)
+    end
+
+    add_column :teams, :personal_owner_id, :integer, index: { unique: true }
+
+    User::Human.all.find_each do |user|
+      user.create_personal_team!
+    end
+  end
+
+  def down
+    User::Human.all.find_each do |user|
```

Comment:
do not destroy personal teams here. that could lead to data loss !
just remove type column and fk_id should be fine